elasticsearch/macos/opendistro-tar-install.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+ES_HOME=`dirname $(realpath $0)`; cd $ES_HOME
```

Comment:
Is this the same file as tar-install.sh? Even if yes rename it to install macos-install.sh
Also I think we discussed realpath doesnot work by default on macos

Reply:
I think we are using the artifact name for the install script, which is still true here. Adding macos in the name will be inconsistent, as then we need to change the install script in Linux to linux-install.sh.

Reply:
Sure. Just `install.sh` would also have worked even for linux tarball but okay.